templates/zerver/api/outgoing-webhooks.md

```diff
@@ -103,15 +103,38 @@ A correctly implemented endpoint will do the following:
 * For a failed request, the endpoint should return data on the
   error.
 
-### Example payload
+### Example request payloads
+
+This is an example of the JSON payload that the Zulip server will `POST`
+to your server:
 
 {generate_code_example|zulip-outgoing-webhook-payload|fixture}
 
+### Example response payloads
+
+Here's an example of the JSON your server should respond with if
+you would not like to send a response message:
+
+```
+{
+    "response_not_required": true
+}
+```
+
+Here's an example of the JSON your server should respond with if
+you would like to send a response message:
+
+```
+{
+    "response_string": "Hey, we just received something from Zulip!"
+}
+```
+
 ## Slack-format webhook format
 
 This interface translates the Zulip's outgoing webhook's request into
-Slack's outgoing webhook request.  Hence the outgoing webhook bot
-would be able to post data to URLs which support Slack's outgoing
+Slack's incoming webhook request.  Hence the outgoing webhook bot
```

Comment:
The wording choices of outgoing vs. incoming are inherently confusing here, since any webhook format has an outgoing side (Zulip/Slack) and incoming side (bot servers, which support either Zulip/Slack format).  I think the original wording is more correct here.  If you change it, I'd say a "Slack-style bot server's incoming request" or something like that.